information-schema/information-schema-placement-rules.md

```diff
@@ -0,0 +1,80 @@
+---
+title: PLACEMENT_RULES
+summary: Learn the `PLACEMENT_RULES` information_schema table.
+---
+
+# PLACEMENT_RULES
+
+The `PLACEMENT_RULES` table provides information on all explicitly configured [placement rules](/placement-rules.md). This includes both placement policies, and directly attached rules.
+
+{{< copyable "sql" >}}
+
+```sql
+USE information_schema;
+DESC placement_rules;
+```
+
+```sql
++----------------------+--------------+------+------+---------+-------+
+| Field                | Type         | Null | Key  | Default | Extra |
++----------------------+--------------+------+------+---------+-------+
+| POLICY_ID            | bigint(64)   | NO   |      | NULL    |       |
+| CATALOG_NAME         | varchar(512) | NO   |      | NULL    |       |
+| POLICY_NAME          | varchar(5)   | YES  |      | NULL    |       |
+| SCHEMA_NAME          | varchar(5)   | YES  |      | NULL    |       |
+| TABLE_NAME           | varchar(5)   | YES  |      | NULL    |       |
```

Comment:
I think for the name fields, the length should be 64
```
mysql> desc tables;
+----------------------------+---------------+------+------+-------------+-------+
| Field                      | Type          | Null | Key  | Default     | Extra |
+----------------------------+---------------+------+------+-------------+-------+
| TABLE_CATALOG              | varchar(512)  | YES  |      | NULL        |       |
| TABLE_SCHEMA               | varchar(64)   | YES  |      | NULL        |       |
| TABLE_NAME                 | varchar(64)   | YES  |      | NULL        |       |
| TABLE_TYPE                 | varchar(64)   | YES  |      | NULL        |       |
| ENGINE                     | varchar(64)   | YES  |      | NULL        |       |
| VERSION                    | bigint(21)    | YES  |      | NULL        |       |
| ROW_FORMAT                 | varchar(10)   | YES  |      | NULL        |       |
| TABLE_ROWS                 | bigint(21)    | YES  |      | NULL        |       |
| AVG_ROW_LENGTH             | bigint(21)    | YES  |      | NULL        |       |
| DATA_LENGTH                | bigint(21)    | YES  |      | NULL        |       |
| MAX_DATA_LENGTH            | bigint(21)    | YES  |      | NULL        |       |
| INDEX_LENGTH               | bigint(21)    | YES  |      | NULL        |       |
| DATA_FREE                  | bigint(21)    | YES  |      | NULL        |       |
| AUTO_INCREMENT             | bigint(21)    | YES  |      | NULL        |       |
| CREATE_TIME                | datetime      | YES  |      | NULL        |       |
| UPDATE_TIME                | datetime      | YES  |      | NULL        |       |
| CHECK_TIME                 | datetime      | YES  |      | NULL        |       |
| TABLE_COLLATION            | varchar(32)   | YES  |      | utf8mb4_bin |       |
| CHECKSUM                   | bigint(21)    | YES  |      | NULL        |       |
| CREATE_OPTIONS             | varchar(255)  | YES  |      | NULL        |       |
| TABLE_COMMENT              | varchar(2048) | YES  |      | NULL        |       |
| TIDB_TABLE_ID              | bigint(21)    | YES  |      | NULL        |       |
| TIDB_ROW_ID_SHARDING_INFO  | varchar(255)  | YES  |      | NULL        |       |
| TIDB_PK_TYPE               | varchar(64)   | YES  |      | NULL        |       |
| TIDB_PLACEMENT_POLICY_NAME | varchar(64)   | YES  |      | NULL        |       |
| TIDB_DIRECT_PLACEMENT      | varchar(5)    | YES  |      | NULL        |       |
+----------------------------+---------------+------+------+-------------+-------+
```

Reply:
I think you are right. That's a bug.

Reply:
https://github.com/pingcap/tidb/issues/29482

Reply:
I will resolve this for now since it is "correct" according to the source code, but we can fix it when we close the bug.